Dockerfile

```diff
@@ -16,16 +16,22 @@ COPY controllers/ controllers
 COPY api/ api
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o kraan-controller main/main.go
+RUN mkdir bin
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o bin/kraan-controller main/main.go
 
-FROM alpine:3.12
-WORKDIR /
-RUN apk add curl
-COPY --from=builder /workspace/kraan-controller /usr/local/bin/
+RUN apt install -y curl
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.12/bin/linux/amd64/kubectl
-RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
 RUN chmod +x ./kubectl
-RUN mv ./kubectl /usr/local/bin
+RUN mv kubectl bin
+RUN curl -LO https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v3.8.5/kustomize_v3.8.5_linux_amd64.tar.gz
+RUN tar xzf ./kustomize_v3.8.5_linux_amd64.tar.gz
+RUN chmod +x ./kustomize
+RUN mv kustomize bin
+
+FROM alpine:3.12
```

Comment:
I would use scratch or [distroless](https://github.com/GoogleContainerTools/distroless) instead of alpine.

Reply:
Thanks, will check this with Rajan see if he has a preference and if he is happy with scratch or distroless then will make change

Reply:
Ok cool 👍